French translation

Aixiis opened the pull request:
Addition of the first "fr_fr.json" file !

Aixiis committed:
French translation

Addition of the first "fr_fr.json" file !

Kira-NT commented on src/main/resources/assets/sync/lang/fr_fr.json:
```diff
@@ -0,0 +1,24 @@
+
+  "item.sync.sync_core": "Noyau de synchronisation",
+
+  "death.attack.sync.fingerstick": "%s est mort pour faire avancer la science !",
```
Should an exclamation mark really stand one space apart from the text, or is it just a typo?

Aixiis replied:
It is not required, it is just the way I write it, I (or you) can change it !

Kira-NT replied:
> French. Yes, the whole language. In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation) should always have a space preceding them. For example, if you wanted to text your friend about how much you hate being online, you'd say Je déteste être en ligne !

Oh, ok. It seems that I don't remember much from my french lessons :D

Kira-NT reviewed (changes requested):
LGTM, but I have one question

Aixiis committed:
Updated French Translation

Removing an unnecessary space in the "...est most pour faire avancer la science..." sentence

Kira-NT reviewed (approved):
Ok, everything looks nice

Aixiis committed:
Revert "Updated French Translation"

This reverts commit ce70c217bdd7b1b38d8e9dcfaa2db11a70e8923f.
